apps/publish/published_item.py

```diff
@@ -137,6 +137,17 @@ class PublishedItemService(BaseService, HighlightsSearchMixin):
 
     SEQ_KEY_NAME = "published_item_sequence_no"
 
+    def patch(self, id, updates):
+        """
+        FIXME: Temporary workaround until we find out why _id and guid are not in sync
+        when running tests from `superdesk-planning`. Without this, many tests fill fail.
+        """
+        item = self.find_one(req=None, guid=id)
+        if item:
+            id = item[ID_FIELD]
```

Comment:
I could not find the root of this problem unfortunately. I just added this workaround with the full intention of finding why are the `_id` and `guid` out of sync. So far what I've found is that when the `published` item is created, the values are fine, but apparently there is some action happening in between that is modifying the `_id` with an `ObjectId` value.

Reply:
@eos87 The `_id` field on the `published` collection uses an ObjectId. There could be multiple entries for the one item (multiple versions). What we use instead is the `item_id` field, as that is what matches the `_id` or `guid` from the `archive` collection (and other content based collections).

Where specifically in the Planning module is this causing an issue?

Reply:
Removed as we spoke earlier today.